Description: relax minimum DH size to 768 bits for compatibility reasons
Author: Marc Deslauriers <marc.deslauriers@canonical.com>

--- nss-3.21/nss/lib/nss/nssoptions.h.1	2016-01-11 20:30:57.876727166 +0300
+++ nss-3.21/nss/lib/nss/nssoptions.h	2016-01-11 20:31:51.581850789 +0300
@@ -16,6 +16,6 @@
 /* 1023 to avoid cases where p = 2q+1 for a 512-bit q turns out to be
  * only 1023 bits and similar.  We don't have good data on whether this
  * happens because NSS used to count bit lengths incorrectly. */
-#define SSL_DH_MIN_P_BITS 1023
+#define SSL_DH_MIN_P_BITS 768
 #define SSL_DSA_MIN_P_BITS 1023
 
